### PR TITLE
Fix charging profile module typos

### PR DESCRIPTION
**Pull request by jovankricka**

### CODE REVIEW

File: images/sd_charging_profile_clear.svg [hunk 1]
```diff
@@ -1 +1 @@
...
...#A80036" points="714,208.2578,704,212.2578,714,216.2578,710,212.2578" style="stroke: #A80036; stroke-width: 1.0;"/><line style="stroke: #A80036; stroke-width: 1.0;" x1="708" x2="875" y1="212.2578" y2="212.2578"/><text fill="#000000" font-family="sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="149" x="720" y="207.1919">ClearChargingProfile.conf</text><polygon fill="#A80036" points="82,252.5234,72,256.5234,82,260.5234,78,256.5234" style="stroke: #A80036; stroke-width: 1.0;"/><line style="stroke: #A80036; stroke-width: 1.0;" x1="76" x2="692" y1="256.5234" y2="256.5234"/><text fill="#000000" font-family="sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="193" x="88" y="236.3247">POST https://client.com/12345</text><text fill="#000000" font-family="sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="244" x="88" y="251.4575">ClearProfileResult(result = ACCEPTED)</text><polygon fill="#A80036" points="686,266.5234,696,270.5234,686,274.5234,690,270.5234" style="stroke: #A80036; stroke-width: 1.0;"/><line style="stroke: #A80036; stroke-width: 1.0; stroke-dasharray: 2.0,2.0;" x1="66" x2="692" y1="270.5234" y2="270.5234"/></g></svg>
```
Comment (RobertdeLeeuw-IHomer) [inline images/sd_charging_profile_clear.svg]: Please change the PlantUML file, that is the source from which the diagram is created. This SVG file will be overwritten during the generation of the PDF.

File: images/sd_charging_profile_updated_via_msp.svg [hunk 2]
```diff
@@ -1 +1 @@
...Length="145" x="279.5" y="164.0591">ActiveChargingProfile()</text><polygon fill="#A80036" points="246.5,179.125,236.5,183.125,246.5,187.125,242.5,183.125" style="stroke: #A80036; stroke-width: 1.0;"/><line style="stroke: #A80036; stroke-width: 1.0; stroke-dasharray: 2.0,2.0;" x1="240.5" x2="614.5" y1="183.125" y2="183.125"/><polygon fill="#A80036" points="988.5,223.3906,998.5,227.3906,988.5,231.3906,992.5,227.3906" style="stroke: #A80036; stroke-width: 1.0;"/><line style="stroke: #A80036; stroke-width: 1.0;" x1="625.5" x2="994.5" y1="227.3906" y2="227.3906"/><text fill="#000000" font-family="sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="356" x="632.5" y="207.1919">PUT https://www.server.com/ocpi/2.2/chargingprofiles/15</text><text fill="#000000" font-family="sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="145" x="664.5" y="222.3247">ActiveChargingProfile()</text><polygon fill="#A80036" points="631.5,237.3906,621.5,241.3906,631.5,245.3906,627.5,241.3906" style="stroke: #A80036; stroke-width: 1.0;"/><line style="stroke: #A80036; stroke-width: 1.0; stroke-dasharray: 2.0,2.0;" x1="625.5" x2="1004.5" y1="241.3906" y2="241.3906"/></g></svg>
```
Comment (RobertdeLeeuw-IHomer) [inline images/sd_charging_profile_updated_via_msp.svg]: Please change the PlantUML file, that is the source from which the diagram is created. This SVG file will be overwritten during the generation of the PDF.

File: images/sd_charging_profile_clear.svg [hunk 3]
```diff
@@ -1 +1 @@
...="#A80036" points="714,208.2578,704,212.2578,714,216.2578,710,212.2578" style="stroke: #A80036; stroke-width: 1.0;"/><line style="stroke: #A80036; stroke-width: 1.0;" x1="708" x2="875" y1="212.2578" y2="212.2578"/><text fill="#000000" font-family="sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="149" x="720" y="207.1919">SetChargingProfile.conf</text><polygon fill="#A80036" points="82,252.5234,72,256.5234,82,260.5234,78,256.5234" style="stroke: #A80036; stroke-width: 1.0;"/><line style="stroke: #A80036; stroke-width: 1.0;" x1="76" x2="692" y1="256.5234" y2="256.5234"/><text fill="#000000" font-family="sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="193" x="88" y="236.3247">POST https://client.com/12345</text><text fill="#000000" font-family="sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="244" x="88" y="251.4575">ClearProfileResult(result = ACCEPTED)</text><polygon fill="#A80036" points="686,266.5234,696,270.5234,686,274.5234,690,270.5234" style="stroke: #A80036; stroke-width: 1.0;"/><line style="stroke: #A80036; stroke-width: 1.0; stroke-dasharray: 2.0,2.0;" x1="66" x2="692" y1="270.5234" y2="270.5234"/></g></svg>
```
Comment (RobertdeLeeuw-IHomer) [inline images/sd_charging_profile_clear.svg]: Please change the PlantUML file, that is the source from which the diagram is created. This SVG file will be overwritten during the generation of the PDF.

File: mod_charging_profiles.asciidoc [hunk 4]
```diff
@@ -259,8 +259,8 @@ The eMSP forwards this response to the SCSP.
 The CPO send the clear requests to the Charge Point.
 When the CPO receives a response from the Charge Point,
 that result is send to the eMSP by call the <<mod_charging_profiles_msp_post_method,POST>> method,
-on the URL provided by the eMSP in the <<mod_charging_profiles_cpo_put_method,PUT>> request of the eMSP.
-The eMSP forwards this result to the the URL provided by the SCSP in the <<mod_charging_profiles_cpo_put_method,PUT>> request of the SCSP,
+on the URL provided by the eMSP in the <<mod_charging_profiles_cpo_get_method,GET>> request of the eMSP.
```
Comment (RobertdeLeeuw-IHomer) [inline mod_charging_profiles.asciidoc]: You are right, PUT is not the method called to clear a charging profile, but it is also not GET, it should be DELETE.

Comment (jovankricka) [reply mod_charging_profiles.asciidoc]: Fixed.

File: images/sd_charging_profile_clear.svg (hunk 3 above)
Comment (jovankricka) [reply images/sd_charging_profile_clear.svg]: Done. Reverted svg changes.

File: images/sd_charging_profile_updated_via_msp.svg (hunk 2 above)
Comment (jovankricka) [reply images/sd_charging_profile_updated_via_msp.svg]: Done.

File: images/sd_charging_profile_clear.svg (hunk 1 above)
Comment (jovankricka) [reply images/sd_charging_profile_clear.svg]: Done.

**Comment by RobertdeLeeuw-IHomer** [comment]
Thank you, I have also seen some other typos that I will fix.

**Pull request merged**